src/app/modules/group/components/group-remove-button/group-remove-button.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Group } from '../../http-services/get-group-by-id.service';
+import { catchError, map, tap } from 'rxjs/operators';
+import { BehaviorSubject, merge, Observable, of } from 'rxjs';
+import { GetGroupChildrenService, GroupChild } from '../../http-services/get-group-children.service';
+import { ConfirmationService } from 'primeng/api';
+import { ActionFeedbackService } from '../../../../shared/services/action-feedback.service';
+import { GroupDeleteService } from '../../services/group-delete.service';
+import { Router } from '@angular/router';
+
+type GroupChildrenState = 'loading' | 'hasChildren' | 'empty' | 'error';
+
+@Component({
+  selector: 'alg-group-remove-button',
+  templateUrl: './group-remove-button.component.html',
+  styleUrls: [ './group-remove-button.component.scss' ]
+})
+export class GroupRemoveButtonComponent implements OnInit {
+  @Input() group!: Group;
+
+  loadingSubject$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
+  loading$: Observable<boolean> = this.loadingSubject$.asObservable();
```

Comment:
1) In my opinion, as this variable in not shared among component/service, it is not useful (it's rather quite polluting the code) to have 2 variables.
2) loading is probably not an appropriate naming as there are 2 loading in this component (the fetching of info, and waiting for deletion). May be called `deletionInProgress$`

Reply:
Fixed